features/plugins.feature

```diff
@@ -35,3 +35,22 @@ Feature: Configuring and using plugins
     And I should see "Whatever" in "_site/index.html"
     And the "_site/test.txt" file should exist
     And I should see "this is a test" in "_site/test.txt"
+
+  Scenario: Plugin is enabled
+    Given I have an "index.html" page that contains "{% if site.plugins contains "jekyll_test_plugin" %} plugin loaded {% endif %}"
```

Comment:
Apologies, i thought i did test that manually, but turns out i didn't... This is indeed not working. The only solution i'd see is to override [`Liquid::Block#unknown_tag`](https://github.com/Shopify/liquid/blob/1370a102c95b83aeb03d9b7290b269435bc66753/lib/liquid/block.rb#L26-L38) but i'm afraid there is no logic for that which would make sense?